cpp/example_code/ses/README.md

```diff
@@ -0,0 +1,108 @@
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+
+
+Before using the code examples, first complete the installation and setup steps
+for [Getting started](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html) in the AWS SDK for
+C++ Developer Guide.
+This section covers how to get and build the SDK, and how to build your own code by using the SDK with a
+sample Hello World-style application.
+
+Next, for information on code example structures and how to build and run the examples, see [Getting started with the AWS SDK for C++ code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started-code-examples.html).
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Create a receipt filter](create_receipt_filter.cpp#L26) (`CreateReceiptFilter`)
+* [Create a receipt rule](create_receipt_rule.cpp#L27) (`CreateReceiptRule`)
+* [Create a receipt rule set](create_receipt_rule_set.cpp#L23) (`CreateReceiptRuleSet`)
+* [Create an email template](create_template.cpp#L24) (`CreateTemplate`)
+* [Delete a receipt filter](delete_receipt_filter.cpp#L23) (`DeleteReceiptFilter`)
+* [Delete a receipt rule](delete_receipt_rule.cpp#L23) (`DeleteReceiptRule`)
+* [Delete a rule set](delete_receipt_rule_set.cpp#L23) (`DeleteReceiptRuleSet`)
+* [Delete an email template](delete_template.cpp#L23) (`DeleteTemplate`)
+* [Delete an identity](delete_identity.cpp#L23) (`DeleteIdentity`)
+* [Get an existing email template](get_template.cpp#L24) (`GetTemplate`)
+* [List identities](list_identities.cpp#L23) (`ListIdentities`)
+* [List receipt filters](list_receipt_filters.cpp#L25) (`ListReceiptFilters`)
+* [Send email](send_email.cpp#L28) (`SendEmail`)
+* [Send templated email](send_templated_email.cpp#L25) (`SendTemplatedEmail`)
+* [Update an email template](update_template.cpp#L24) (`UpdateTemplate`)
+* [Verify an email identity](verify_email_identity.cpp#L23) (`VerifyEmailIdentity`)
+
+### Cross-service examples
+
+Sample applications that work across multiple AWS services.
+
+* [Create an Aurora Serverless work item tracker](../../example_code/cross-service/serverless-aurora)
+
+## Run the examples
+
+### Instructions
+
+
+<!--custom.instructions.start-->
+<!--custom.instructions.end-->
+
+
+
+### Tests
+
+⚠ Running tests might result in charges to your AWS account.
+
+
+
+```sh
+   cd <BUILD_DIR>
+   cmake <path-to-root-of-this-source-code> -DBUILD_TESTS=ON
+   make
+   ctest
+```
+
+
+<!--custom.tests.start-->
+<!--custom.tests.end-->
+
+## Additional resources
+
+* [Amazon SES Developer Guide](https://docs.aws.amazon.com/ses/latest/dg/Welcome.html)
+* [Amazon SES API Reference](https://docs.aws.amazon.com/ses/latest/APIReference/Welcome.html)
+* [SDK for C++ Amazon SES reference](https://sdk.amazonaws.com/cpp/api/LATEST/aws-cpp-sdk-ses/html/annotated.html)
```

Comment:
```suggestion
* [SDK for C++ Amazon SES reference](https://sdk.amazonaws.com/cpp/api/LATEST/aws-cpp-sdk-sesv2/html/annotated.html)
```

Reply:
Actually, sesv2 is a different api. The correct link is "aws-cpp-sdk-email". Thanks for catching that.